src/main/java/Pages/AdminTablePage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import com.codeborne.selenide.SelenideElement;
+import com.codeborne.selenide.Condition;
+import static com.codeborne.selenide.Selenide.*;
+import static org.openqa.selenium.By.*;
+import com.codeborne.selenide.Condition;
+
+public class AdminTablePage {
+
+    private String markerXPath = "//h2[contains(text(),'Список администраторов')]";
+    private String createButtonCSS = "btn.btn-green";
+    private String newAdmin = "testFirst testLast";
+    private String newAdminLoc = "//a[contains(text(),'testFirst testLast')]";
+    private String deleteScript = "http://forge.f-d.com.ua/sections/administrator/admin/delete/";
+    public SelenideElement getAdminTableMarker() {
+        return $(xpath(markerXPath));
+    }
+
+    public void checkNewAdmin() {
+        $(xpath(newAdminLoc)).waitUntil(Condition.visible, 10000);
+    }
+
+    public void deleteNewAdmin() {
+        String AdminLink = $(xpath(newAdminLoc)).getAttribute("href");
+        String AdminId = AdminLink.substring(AdminLink.lastIndexOf("/") + 1);
+        open(deleteScript + AdminId);
```

Comment:
if all we are trying to do is send http call - let's not do it through browser.

it's not a part of the page - let's write some Service Class and send request through some http request library. 
easiest way will be RestAssured. we will also use it for API testing anyway